php/common.php

```diff
@@ -233,6 +233,29 @@ function get_settings($filename = "")
         $ini->updateFile();
     }
 
+    if ($user_version < 3) {
+        // Парсим опцию исключения из отчётов
+        $excludeForumsIDs = $ini->read("reports", "exclude", "");
+        $excludeForumsIDs = array_filter(explode(",", $excludeForumsIDs));
+        $excludeForumsIDs = array_unique($excludeForumsIDs);
+        $ini->write("reports", "exclude", "");
+
+        if (count($excludeForumsIDs)) {
+            foreach ($excludeForumsIDs as $forum_id) {
+                $config['subsections'][$forum_id]['exclude'] = 1;
```

Comment:
A bit hard to do a real testing for me now, let me just ask here... What would happen if there is no `$forum_id` in `$config['subsections']` yet? Does it cause error?

Reply:
If just `$forum_id` is not set, then will be created new section in `$config['subsections']`

```

[1234]
exclude="1"

[4321]
exclude="1"
```

Probably, its not best way, but why was in `explode` section an incorrect forum_id in the first place?

Reply:
Ah, good then!

> Probably, its not best way, but why was in `explode` section an incorrect forum_id in the first place?

An incorrect ID could have been because it is manually written by the user and has not been controlled by TLO at all.

Reply:
Добавил проверку наличия ид исключаемой секции в секциях. Не будут создаваться лишние блоки в конфиге, если в `exclude` написана лишняя инфа.